antiplag/tasks.py

```diff
@@ -95,37 +87,37 @@ def compare_documents(
                 # returns percentage representing how similar docs are
                 similarity = text_comparison(doc.text, similar_doc["text"])
 
-                result_similarity += similarity
+                result_similarity += similarity[0][1]
```

Comment:
What if we returned dictionary instead of array? When I read this code I hardly get what `[0][1]` indices represent.